lib/validate_encrypt_utils.pm

```diff
@@ -63,13 +110,27 @@ sub verify_number_of_encrypted_devices {
           Dumper($actual_number));
 }
 
+=head2 verify_cryptsetup_message 
+
+  verify_cryptsetup_message($expected_message, $actual_message);
+
+  C<$expected_message>: A string with the expected status message of a device
+  C<actual_message>: A string with the actual status message of a device
```

Comment:
`s/actual_message/$actual_message/`. Please fix this issue for other perlpod documentation. `$`, as well as `@` or `%` show which data type it is, so makes sense to specify them.

Reply:
to be honest, i checked around in the repo to see how others are doing it and i got confused. So i use your approach trying to show the data types. But then i found cases that i did not know how it should be? for example, ` C<$expected_properties>` is a hash ref. not easy to tell from `$` what is the type. I took a look also to the pod documentation and applied some changes.

Reply:
Main problem I see is that you mix both approaches. And if you put scalar context, with proper text description the reader will see that it's hash reference. If you insist, you can keep just names, but then do it consistently in the whole file ;)